include/hyprutils/utils/FileDescriptor.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+namespace Hyprutils {
+    namespace Utils {
+        class CFileDescriptor {
+          public:
+            CFileDescriptor() = default;
+            explicit CFileDescriptor(int fd);
+            CFileDescriptor(CFileDescriptor&&);
```

Comment:
forgot const (see rule of 3 5 0 https://en.cppreference.com/w/cpp/language/rule_of_three )

Reply:
added const to explicit constructor if thats the one you meant

Reply:
yes

Reply:
no

Reply:
yes nvm